README.md

```diff
@@ -1,3 +1,4 @@
 # zomatoscrape
 This is first repository
 This is my first commit
+I am creating merge request
```

Comment:
Practice change.
Write anything more, add a line more and push it again to same branch